runner.py

```diff
@@ -35,8 +35,8 @@ def run_single_experiment(model_name: str, seed: int | None, hyperparams: dict):
     set_seed(seed)
 
     model = get_callable_from_string(hyperparams["model"])
-    floatnet = model(pretrained=False)
-    bitnet = model(pretrained=False)
+    floatnet = model(weights=None)
```

Comment:
The parameter `weights=None` is confusing. What does that mean?

Reply:
It's the new standard way instead of `pretrained=False`
Not my fault